html_challenge.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+    <h1>
+        Sundial
+    </h1>
+    <p>
+
+    From Wikipedia, the <em> free</em> encyclopedia
+    </p>
+
+     <strong>a sundial measures time by the position of the sun.</strong> The most commonly seen designs, such as the
+'ordinary' or standard garden sundial, cast a shadow on a flat surface marked with the hours of
+the day. As the position of the sun changes, the time indicated by the shadow changes. However,
+sundials can be designed for any surface where a fixed object casts a predictable shadow.
+<p>
```

Comment:
Reminder, just like code, you should have proper indentation if a html tag is inside another one, this makes it easier to read.